advanced_functionality/pytorch_extend_container_train_deploy_bertopic/BERTtopic_extending_container.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>Only put the title of the notebook as H1 - other titles (i.e. Extending our PyTorch containers) should be H2 or less. It affects how the notebook is rendered in ReadTheDocs. Do this for the entire notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done, thanks